sky/utils/common_utils.py

```diff
@@ -24,20 +24,20 @@
 
 logger = sky_logging.init_logger(__name__)
 
-_run_id = None
+_usage_run_id = None
 
 
-def get_run_id():
+def get_usage_run_id():
```

Comment:
For this and the other functions, why don't we annotate their return types?
```suggestion
def get_usage_run_id() -> str:
```

Reply:
Good catch. Fixed